sdk/src/test/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/connector/ApiConnectorTest.java

```diff
@@ -150,6 +154,16 @@ private void assertFirstMerchantOrderResult(MerchantOrderResult actual) {
         assertThat(actual.getPaidAmount().getAmountInCents(), is(0L));
         assertThat(actual.getTotalAmount().getCurrency(), is(EUR));
         assertThat(actual.getTotalAmount().getAmountInCents(), is(599L));
+        assertThat(actual.getTransactionInfo().get(0).getId(), is("1"));
+        assertThat(actual.getTransactionInfo().get(0).getPaymentBrand(), is(PaymentBrand.IDEAL));
+        assertThat(actual.getTransactionInfo().get(0).getType(), is(TransactionType.PAYMENT));
+        assertThat(actual.getTransactionInfo().get(0).getStatus(), is(TransactionStatus.SUCCESS));
```

Comment:
The example might be a bit confusing as you can't have an order in state CANCELLED an a successful transaction at the same time. Possible solutions would be to change the transaction status to e.g. FAILURE,or remove any transactions from this order result so we also test this specific case.